Makefile

```diff
@@ -42,14 +42,17 @@ install:
 
 all: gen-project gendoc
 %.yaml: gen-project
-deploy: all mkd-gh-deploy
+deploy: gendoc mkd-gh-deploy
```

Comment:
Why rerun artifact generation step once again while doing `deploy`?

We just need the HTML docs in the right directory for deploying, so we can make gendoc a dependency on the deploy target